jaeger_client/utils.py

```diff
@@ -22,25 +22,19 @@
 import struct
 import time
 
-from .metrics import Metrics
-
 
 class ErrorReporter(object):
     """
     Reports errors by emitting metrics, and if logger is provided,
     logging the error message once every log_interval_minutes
     """
 
-    def __init__(self, metrics, logger=None, log_interval_minutes=15):
+    def __init__(self, logger=None, log_interval_minutes=15):
```

Comment:
please keep the metrics argument. For some reason we allowed reporter to be created externally, so it's a public API now.